src/database.types.ts

```diff
@@ -0,0 +1,357 @@
+export type Json =
+  | string
+  | number
+  | boolean
+  | null
+  | { [key: string]: Json | undefined }
+  | Json[];
+
+export interface Database {
+  graphql_public: {
```

Comment:
Another thing I'm confused by is that I have differences in this file without really touching anything in Supabase. Maybe something is off in our configuration but I'm not sure.

Reply:
Resolved in #36